libraries/AP_BoardConfig/AP_BoardConfig.cpp

```diff
@@ -246,5 +255,10 @@ void AP_BoardConfig::init()
 #elif CONFIG_HAL_BOARD == HAL_BOARD_VRBRAIN
     /* configure the VRBRAIN driver for the right number of PWMs */
 
-#endif    
+#endif
+
+    // let the HAL know the target temperature. We pass a pointer as
+    // we want the user to be able to change the parameter without
+    // rebooting
```

Comment:
Ok. Now I got why it is a pointer: when setting a parameter we just set the value, there's no special handler function.